### ISSUE TITLE
[BUG] Adding a long caption throws an error

### ISSUE THREAD

**Issue by aditeyabaral** [aditeyabaral/spongebob-as-a-service#5]
Adding an extremely long caption throws `OSError: cannot open resource`.  This seems to indicate that it does not find the font file for some reason. This issue is a top priority.

### How to generate the error?

Copy paste this into your browser - 
`https://spongebob-service.herokuapp.com//hello friends/chai peelo peelo peelo peelo peelo peelo peelo peelo peelo peelo peelo peelo peelo peelo peelo peelo`

### Error Stack Trace
```bash
[2021-02-06 22:18:16,799] ERROR in app: Exception on /hello friends/chai peelo peelo peelo peelo peelo peelo peelo peelo peelo peelo peelo peelo peelo peelo peelo peelo [GET]
Traceback (most recent call last):
  File "/home/aditeya/Desktop/spongebob-as-a-service/saas/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/aditeya/Desktop/spongebob-as-a-service/saas/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/aditeya/Desktop/spongebob-as-a-service/saas/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/aditeya/Desktop/spongebob-as-a-service/saas/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/aditeya/Desktop/spongebob-as-a-service/saas/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/aditeya/Desktop/spongebob-as-a-service/saas/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app/saas.py", line 24, in meme
    filename = generateMeme(captions)
  File "/home/aditeya/Desktop/spongebob-as-a-service/app/utils.py", line 39, in generateMeme
    addText(img, position, caption)
  File "/home/aditeya/Desktop/spongebob-as-a-service/app/utils.py", line 66, in addText
    font = ImageFont.truetype("impact.ttf", fontSize)
  File "/home/aditeya/Desktop/spongebob-as-a-service/saas/lib/python3.8/site-packages/PIL/ImageFont.py", line 852, in truetype
    return freetype(font)
  File "/home/aditeya/Desktop/spongebob-as-a-service/saas/lib/python3.8/site-packages/PIL/ImageFont.py", line 849, in freetype
    return FreeTypeFont(font, size, index, encoding, layout_engine)
  File "/home/aditeya/Desktop/spongebob-as-a-service/saas/lib/python3.8/site-packages/PIL/ImageFont.py", line 209, in __init__
    self.font = core.getfont(
OSError: cannot open resource
```

**Comment by rohangrge** [comment]
I want to try this using parameters instead of having it as routes ,are you open or that?

**Comment by aditeyabaral** [comment]
> I want to try this using parameters instead of having it as routes ,are you open or that?

Are you referring to using POST requests?

**Comment by aditeyabaral** [comment]
> I want to try this using parameters instead of having it as routes ,are you open or that?

I don't see any added advantage by using parameters. 
However, do try it out and see if it can solve the issue.

**Comment by rohangrge** [comment]
Can you tell me how to run this,getting certain errors.

**Comment by aditeyabaral** [comment]
Solved in #6
